e2e/deleteapi_test.go

```diff
@@ -34,10 +34,9 @@ func TestDeleteAPI(t *testing.T) {
 	// apiserver to go down the next step will return sucess before the
 	// apiserver is ever destroyed.
 	waitDestroy := func() error {
-		// only checking api being down , specific function not important
+		// only checking api being down, specific function not important
 		_, err := client.Discovery().ServerVersion()
-
-		if err == nil {
+		if err != nil {
```

Comment:
`err == nil` is what we want here since, we are specifically looping until we can no longer contact the apiserver as evidence that it was successfully destroyed. Maybe there is a more clear way to write it so it doesn't look like a typo though.